### PR TITLE
Rename variables at Lock.swift

### PR DESCRIPTION
**Pull request by unnamedd**
The intention of this PR is to rename two simple variables on the file Lock.swift to give more context to them.

### CODE REVIEW

**Comment by unnamedd** [comment]
@swift-ci please smoke test

File: Sources/Basic/Lock.swift [hunk 1]
```diff
@@ -58,8 +58,8 @@ public final class FileLock {
     /// Note: This method can throw if underlying POSIX methods fail.
     public func lock() throws {
       #if os(Windows)
-        if h == nil {
-            let h = lockFile.pathString.withCString(encodedAs: UTF16.self, {
+        if handle == nil {
+          let h = lockFile.pathString.withCString(encodedAs: UTF16.self, {
```
Comment (jmittert) [inline Sources/Basic/Lock.swift]: Thanks! Minor thing: can you fix the indentation here?

Comment (unnamedd) [reply Sources/Basic/Lock.swift]: Yeah, sure! ;)

**Comment by aciidb0mb3r** [comment]
@swift-ci smoke test

**Comment by aciidb0mb3r** [comment]
@swift-ci please smoke test

**Pull request merged**